src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -503,4 +533,74 @@ private function hasProductWithNameAndShortDescription(array $products, string $
 
         return false;
     }
+
+    private function hasProductOptionWithNameAndValue(string $optionValue, string $optionName): bool
+    {
+        $response = $this->client->getLastResponse();
+        $productOptions = $this->responseChecker->getValue($response, 'options');
+
+        foreach($productOptions as $option){
```

Comment:
```suggestion
        foreach ($productOptions as $option) {
```
and what is under `$option`? IRI? Change the name to be more clear, if it is an IRI -> `$optionIri`